subcommands/waves/cmd.go

```diff
@@ -0,0 +1,23 @@
+package waves
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/foundriesio/fioctl/client"
+	"github.com/foundriesio/fioctl/subcommands"
+)
+
+var api *client.Api
+
+var cmd = &cobra.Command{
+	Use:   "wave",
+	Short: "Manage factory's waves",
```

Comment:
Capitalize the F in factories.

Reply:
ack

Reply:
hmm... we have it lowercase in other places:
```
Available Commands:
  config      Manage configuration common to all devices in a factory
  devices     Manage devices registered to a factory
  help        Help about any command
  keys        Manage keys in use by your factory fleet
  login       Access Foundries.io services with your client credentials
  secrets     Manage secret crendentials configured in a factory
  status      Get dashboard view of a factory and its devices
  targets     Manage factory's TUF targets
  users       List users with access to a factory
  version     Show version information of this tool.
  wave        Manage factory's waves
```

Reply:
Ugh. keep it like this for now. the first usage statement i looked at was different. so there's inconsistency that has nothing to do with you.